doc/images/stimulation-time-diagram.svg

```diff
@@ -0,0 +1,260 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
This is added but not used anywhere. May be I am missing something here ?

Reply:
I'm just thinking we  whoudl ahve the svg somewhere to further edit and improve it...